client/my-sites/patterns/components/readymade-template-preview/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { MShotsImage } from '@automattic/onboarding';
+import { ReadymadeTemplate } from 'calypso/my-sites/patterns/types';
+
+export const ReadymadeTemplatePreview = ( {
+	readymadeTemplate,
+}: {
+	readymadeTemplate: ReadymadeTemplate;
+} ) => {
+	const mShotsOptions = {
+		vpw: 1200,
+		vph: 1680,
+		w: 1200,
+		screen_height: 1680,
+	};
+
+	return (
+		<div className="readymade-template-preview">
+			<MShotsImage url={ readymadeTemplate.previewUrl + '&t=2' } alt="" options={ mShotsOptions } />
```

Comment:
The `t=2` param here is intentional. Once D157805-code is shipped, I'll remove it so we can easily generate new mShots